src/content/pages/backpacking-checklist.md

```diff
@@ -1,5 +1,110 @@
 ---
-title: "Backpacking checklist"
+title: "Backpacking Checklist"
 description: "this is meta description"
 draft: false
----
\ No newline at end of file
+--- ### Core Backpacking Needs
+Pack with padded hip belt
+
+ Waterproof Pack cover – (REQUIRED)
+
+ 6-10 1 Gallon “Hefty” bags for packing clothes
+
+ 2 Plastic kitchen garbage bags to wrap sleeping bag in
+
+ 2-3 32 ounce wide mouth water bottles
+
+ 1-16 ounce bottle or cup (for mixing hot & cold drinks)
+
+ Sleeping Gear
+
+ Sleeping bag (full zip, 30 to 15 degree (recommended)
+
+ Sleeping Pad (closed cell/thermarest recommended)
+
+ Sleeping clothing (worn ONLY in the sleeping bag)
+
+ ### Clothing (hiking apparel)
```

Comment:
Let's make all the words in the titles capitalized, e.g. "Clothing (Hiking Apparel)"